packages/common/core-interfaces/src/provider.ts

```diff
@@ -27,7 +27,7 @@
  *
  * This pattern enables discovery, and delegation in a standard way which is central
  * to FluidObject pattern.
- * @alpha
+ * @public
```

Comment:
instanceof doesn't work across out layer boundaries which sometimes include different javascript contexts, for instance when rendering is done in a same source iframe

Reply:
In general, we've been moving aways from this style of anonymous object feature discovery. ideally we can get to a point where this in a application concern that they implement, rather than something we need to care about